torch/csrc/jit/codegen/cuda/lower_replace_size.cpp

```diff
@@ -282,6 +282,12 @@ void replaceSymbolicSizes(Fusion* fusion) {
   ValReplacementMutator(fusion, tensor_dim_map);
 }
 
+void replaceValue(
+    Fusion* fusion,
+    const std::unordered_map<Val*, Val*>& replacement_map) {
+  ValReplacementMutator(fusion, replacement_map);
+}
```

Comment:
This seems like a generic function that's not specific to size replacement. Why not move this somewhere else? Perhaps, ir_utils?